src/shared/icons/DropDown.tsx

```diff
@@ -0,0 +1,3 @@
+export const DropDown = (): JSX.Element => <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path d="M12 15L8 10H16L12 15Z" fill="#13183A"/>
```

Comment:
We can also use tailwind for styling svg. Please, use custom colors from tailwind.config.js.
Read more: https://tailwindcss.com/docs/fill#setting-the-fill-color

Reply:
It does not work

Reply:
Yes it does: 
Before:
![image](https://github.com/Vladyslav-Samokryk/gift-hub-frontend/assets/62620618/686e10c5-f042-41f3-9201-cd540a75ceca)

![image](https://github.com/Vladyslav-Samokryk/gift-hub-frontend/assets/62620618/83d878a6-7c8d-4f03-b782-43047fd63009)

After:
![image](https://github.com/Vladyslav-Samokryk/gift-hub-frontend/assets/62620618/a0390460-27e7-4abf-8b27-9eb21270b8fa)
![image](https://github.com/Vladyslav-Samokryk/gift-hub-frontend/assets/62620618/2342024d-f8a7-44a9-b732-05c8ff1007a5)